controllers/cloud.redhat.com/pool_controller.go

```diff
@@ -135,6 +137,7 @@ func (r *NamespacePoolReconciler) getPoolStatus(ctx context.Context, pool crd.Na
 
 	var readyNS int
 	var creatingNS int
+	var totalNS int
```

Comment:
I'd argue this isn't necessary here, we already know the total by adding creating and ready

Reply:
"ready" only records namespaces ready to be reserved. It checks the owner reference's UID against the pool's UID:
`
for _, owner := range ns.GetOwnerReferences() {
	if owner.UID == pool.GetUID() {
		switch ns.Annotations["env-status"] {
		case "ready":
			readyNS++
		case "creating":
			creatingNS++
		case "error":
			r.Log.Info("Error status for namespace. Prepping for deletion.", "ns-name", ns.Name)
			errNamespaceList = append(errNamespaceList, ns.Name)
		}
	}
}
`